### PR TITLE
Update unhost email notification copy

### PR DESCRIPTION
**Pull request by hdiniz**
Related https://github.com/opencollective/opencollective/issues/3330
Related https://github.com/opencollective/opencollective/issues/5993

image

### CODE REVIEW

File: templates/emails/collective.unhosted.hbs [hunk 1]
```diff
@@ -6,7 +6,12 @@ Subject: Important: {{{collective.name}}} has been un-hosted by {{{host.name}}}
   Hi {{collective.name}},
   <br /><br />
   Your fiscal host, {{> linkCollective collective=host}}, has un-hosted your Collective.
-  <br/><br/>
+</p>
+<ul style="font-size: 17px; margin-bottom: 1em;">
+  <li>Recurring contributions that cannot be transfered between hosts will be cancelled.</li>
```
Comment (Betree) [inline templates/emails/collective.unhosted.hbs:11]: It's great that we are abstract in the code to lay the ground for future payment methods, but here I think we should be more specific to make sure users understand the consequences.

```suggestion
  <li>PayPal recurring contributions, if any, cannot be transferred. They will be cancelled.</li>
```

**Review by Betree — COMMENTED** [review]
Would be great to share a screenshot preview of the email

**Comment by Betree** [comment]
Also if you need more feedback on the wording feel free to ask in https://opencollective.slack.com/archives/C019YCG6RCY - I'm not the best person to advise on that part :slightly_smiling_face:

File: server/lib/notifications/email.ts [hunk 2]
```diff
@@ -133,10 +133,14 @@ export const notifyByEmail = async (activity: Activity) => {
     case ActivityTypes.COLLECTIVE_EXPENSE_CREATED:
     case ActivityTypes.COLLECTIVE_FROZEN:
     case ActivityTypes.COLLECTIVE_UNFROZEN:
-    case ActivityTypes.COLLECTIVE_UNHOSTED:
     case ActivityTypes.PAYMENT_CREDITCARD_EXPIRING:
       await notify.collective(activity);
       break;
+    case ActivityTypes.COLLECTIVE_UNHOSTED:
+      await notify.collective(activity, {
+        replyTo: 'support@opencollective.com',
```
Comment (Betree) [inline server/lib/notifications/email.ts:141]: Good candidate for the reply to email introduced in https://github.com/opencollective/opencollective-api/pull/8006:
```suggestion
        replyTo: activity.data.host.data?.replyToEmail || 'support@opencollective.com',
```

**Review by Betree — APPROVED** [review]
Looks good to me overall, feel free to merge whenever you feel it's ready

**Comment by kaylarep** [comment]
If I'm not too late, can we possibly:

- [ ] add a hyperlink to https://docs.opencollective.com/help/fiscal-hosts/create-a-fiscal-host#connect-a-stripe-account-and-or-add-a-bank-account under `use your own bank accont`

- [ ]  add a hyperlink to https://docs.opencollective.com/help/independent-collectives/about-independent-collectives under the words `become an Independent Collective` 

Thank you!

**Comment by hdiniz** [comment]
> If I'm not too late, can we possibly:
> 
>     * [ ]  add a hyperlink to https://docs.opencollective.com/help/fiscal-hosts/create-a-fiscal-host#connect-a-stripe-account-and-or-add-a-bank-account under `use your own bank accont`
> 
>     * [ ]   add a hyperlink to https://docs.opencollective.com/help/independent-collectives/about-independent-collectives under the words `become an Independent Collective`
> 
> 
> Thank you!

@kaylarep For sure! I will add them in another pull request

**Pull request merged**